tests/clean_up_mssql_pacemaker.yml

```diff
@@ -32,19 +32,15 @@
       include_role:
         name: fedora.linux_system_roles.firewall
 
-    - name: Remove cluster RPMs
+    - name: Remove mssql-server
       package:
-        name:
-          - pacemaker
-          - pcs
-          - fence-agents-all
-          - resource-agents
+        name: mssql-server
         state: absent
 
-    - name: Remove mssql RPMs
-      shell: dnf remove mssql-* -y
-      changed_when: true
-
     - name: Remove related files
-      shell: rm -rf /var/opt/mssql/ /opt/mssql* /tmp/ansible.*
+      shell: >-
+        rm -rf  /var/opt/mssql \
```

Comment:
you don't need `\` line continuation when using `>-` - this yaml string operator will concatenate all lines into a single line with the lines separated by a single space

Reply:
Thank you, fixed this

Reply:
What you suggest doesn't work.

Running this playbook:
```
    - name: Remove related files
      shell: >-
        rm -rf  /var/opt/mssql
                /opt/mssql
                /tmp/ansible.*
                /var/log/pacemaker/pacemaker.log
      changed_when: true
```
Results in this:
```
fatal: [primary]: FAILED! => {"changed": true, "cmd": "rm -rf  /var/opt/mssql\n        /opt/mssql\n        /tmp/ansible.*\n        /var/log/pacemaker/pacemaker.log", "delta": "0:00:00.009527", "end": "2022-08-16 16:06:49.775887", "msg": "non-zero return code", "rc": 127, "start": "2022-08-16 16:06:49.766360", "stderr": "/bin/sh: line 1: /opt/mssql: No such file or directory\n/bin/sh: line 2: /tmp/ansible.*: No such file or directory\n/bin/sh: line 3: /var/log/pacemaker/pacemaker.log: No such file or directory", "stderr_lines": ["/bin/sh: line 1: /opt/mssql: No such file or directory", "/bin/sh: line 2: /tmp/ansible.*: No such file or directory", "/bin/sh: line 3: /var/log/pacemaker/pacemaker.log: No such file or directory"], "stdout": "", "stdout_lines": []}
```
The resulting command has `\n`s in it. No idea why. I am returning backslashes to break lines.

Reply:
I've found what I did wrong. I must not intend lines if I do not want Jinja to paste `\n`s. This works fine:
```
    - name: Remove related files
      shell: >-
        rm -rf /var/opt/mssql
        /opt/mssql
        /tmp/ansible.*
        /var/log/pacemaker/pacemaker.log
      changed_when: true
```

Reply:
ah, it was the indentation - weird - good to know